index.bs

```diff
@@ -315,6 +317,50 @@ and return {{FileSystemHandleKind/"directory"}} otherwise.
 The <dfn attribute for=FileSystemHandle>name</dfn> attribute must return the [=entry/name=] of the
 associated [=FileSystemHandle/entry=].
 
+### The {{FileSystemHandle/remove()}} method ### {#api-filesystemhandle-remove}
+
+<div class="note domintro">
+  : await |handle| . {{FileSystemHandle/remove()|remove}}()
+  :: Attempts to remove the entry represented by |handle| from the underlying file system.
+
+     For files or directories with multiple hardlinks or symlinks, the entry which is deleted
+     is the entry corresponding to the path which was used to obtain the handle.
```

Comment:
@pwnall This text was snagged from your comment here. Let me know if this needs more elaboration https://github.com/WICG/file-system-access/issues/214#issuecomment-724884965